src/js/clickable-component.js

```diff
@@ -41,9 +46,20 @@ class ClickableComponent extends Component {
       this.controlText(this.options_.controlText);
     }
 
+    const selectClickHandler = () => {
+      if (typeof this.options_.throttle === 'number' || this.options_.throttle === true) {
+        const wait = typeof this.options_.throttle === 'number' ? parseInt(this.options_.throttle, 10) : 50;
```

Comment:
Also, should this be `parseFloat` instead of `parseInt`?

Reply:
I figured since throttle's key condition is `(now - last >= wait) ` where `now - last` will always be a whole number of milliseconds, `parseInt` was most appropriate. 

Looking with fresh eyes, maybe the `parseX(arg-that-might-be-a-number`) should be removed entirely as I really just put it in out of habit and I see `throttle` itself doesn't enforce anything.